src/actors/builtin/storage_miner/storage_miner_actor.id

```diff
@@ -99,7 +99,6 @@ type MinerInfo struct {
 
     // Amount of space in each sector committed to the network by this miner.
     SectorSize              sector.SectorSize
-    RegisteredProof         sector.RegisteredProof
     SealPartitions          UVarint
```

Comment:
I think if this is removed, then `RegisteredProof` needs to be added to the `OnChainSealVerifyInfo` constructed `ProveCommitSector` or somehow otherwise communicated to proofs at verification time. I might be missing the logic of how you are doing this, though. Can you help me understand how/that this is happening?

Reply:
It's already in the OnChainSealVerifyInfo and OnChainElection*VerifyInfo structures which, as I understand it, means the caller will provide it.

Reply:
But I see I may have misread

Reply:
Added it to SectorProveCommitInfo so the caller provides it per-sector.